docs/user-guide/FAQ.md

```diff
@@ -168,6 +169,29 @@ By default, step commands are evaluated with the Bourne shell (`/bin/sh`). You c
 ## How do I speed up time to upload artifacts?
 You can set the environment variable [`SD_ZIP_ARTIFACTS`](./environment-variables#user-configurable) to `true` which will zip artifacts before uploading, provided your cluster admin has set it up properly.
 
+## How do I permalink pipeline jobs and artifacts?
+You can follow the following conventions to permalink your jobs and artifacts for your pipelines:
+
+```
+/pipelines/{PIPELINE_ID}/jobs/{JOB_NAME}/latest?status=SUCCESS
+/pipelines/{PIPELINE_ID}/jobs/{JOB_NAME}/latest?status=FAILURE
+/pipelines/{PIPELINE_ID}/jobs{JOB_NAME}/3
```

Comment:
```suggestion
/pipelines/{PIPELINE_ID}/jobs/{JOB_NAME}/3
```

What does the number `3` refer to?
might be good to explain that too

Reply:
The original intent of a number meant the `n`th build of a given job, for example, 3 means the 3rd build of a given job. Unfortunately, this feature was not implemented. I'm updating the docs to reflect that.